test/test_jit.py

```diff
@@ -12046,6 +12046,9 @@ def func(x):
   return (%1)
 ''')
 
+    def test_generic_list(self):
```

Comment:
nit: rename to make it more clear that this is a custom op test

Reply:
It's under the `TestCustomOperators` class